service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/DocumentUploadedEventHandler.java

```diff
@@ -289,9 +309,25 @@ private List<SupportingEvidenceBundle> getDocuments(
         return newDocs;
     }
 
+    private List<FurtherDocument> getNewUploadedFurtherDocuments(
+        CaseData caseData, CaseData caseDataBefore,
+        BiPredicate<List<FurtherDocument>, FurtherDocument> biPredicate) {
```

Comment:
Think this should have a name to describe what it does better, rather than just "brPredicate"